hardware/1Wire.cpp

```diff
@@ -32,51 +32,38 @@ C1Wire::C1Wire(const int ID, const int sensorThreadPeriod, const int switchThrea
 {
 	m_HwdID = ID;
 
-	// Defaults for pre-existing 1wire instances
-	if (0 == m_sensorThreadPeriod)
```

Comment:
Now "1" (milisecond) is good solution for many people, who need quick reading (especially the status of the switches). 
Previous change in code from another developer, already introduced ms instead of sec.